DQM/EcalMonitorClient/src/PresampleClient.cc

```diff
@@ -108,7 +108,7 @@ namespace ecaldqm {
 
       // Fill Presample Trend plots:
       // Use PedestalByLS which only contains digis from "current" LS
-      float chStatus(sChStatus.getBinContent(getEcalDQMSetupObjects(), id));
+      int chStatus = static_cast<int>(sChStatus.getBinContent(getEcalDQMSetupObjects(), id));
```

Comment:
@iarspider , what is the real type of this status ? why static_cast is needed here?